readme.md

```diff
@@ -1169,6 +1169,9 @@ To reuse a session without reinstalling dependencies use the `-rs` flag instead
 
 # Changelog
 
+## [1.1.5-develop]
```

Comment:
either bump this to 1.1.6 all over or put this change in the 1.1.5-develop below.  your choice.

Reply:
moved to 1.1.5